### ISSUE TITLE
Start page is 10x slower after update to 1.18.0

### ISSUE THREAD

**Issue by somera** [go-gitea/gitea#22299]
### Description

After upgrade to 1.18.0 a see a slowdown loading the start page.

![image](https://user-images.githubusercontent.com/8334250/210137060-f8a94bbd-7740-49c4-813d-e43dfa6f6947.png)

Some optimizations were made in #21031.

This is what I foud at the moment:

I see 2659x this

```
2022/12/31 13:34:28 ...ls/system/setting.go:121:GetSettings() [I] [63b02c54] [SQL] SELECT "id", "setting_key", "setting_value", "version", "created", "updated" FROM "system_setting" WHERE setting_key IN ($1) [picture.disable_gravatar] - 1.144962ms
2022/12/31 13:34:28 ...ls/system/setting.go:121:GetSettings() [I] [63b02c54] [SQL] SELECT "id", "setting_key", "setting_value", "version", "created", "updated" FROM "system_setting" WHERE setting_key IN ($1) [picture.disable_gravatar] - 571.941µs
..
..
..
..
2022/12/31 13:34:31 ...ls/system/setting.go:121:GetSettings() [I] [63b02c54] [SQL] SELECT "id", "setting_key", "setting_value", "version", "created", "updated" FROM "system_setting" WHERE setting_key IN ($1) [picture.enable_federated_avatar] - 508.168µs
2022/12/31 13:34:31 ...ls/system/setting.go:121:GetSettings() [I] [63b02c54] [SQL] SELECT "id", "setting_key", "setting_value", "version", "created", "updated" FROM "system_setting" WHERE setting_key IN ($1) [picture.disable_gravatar] - 423.219µs
2022/12/31 13:34:31 ...ls/system/setting.go:121:GetSettings() [I] [63b02c54] [SQL] SELECT "id", "setting_key", "setting_value", "version", "created", "updated" FROM "system_setting" WHERE setting_key IN ($1) [picture.enable_federated_avatar] - 411.846µs
2022/12/31 13:34:31 ...ls/system/setting.go:121:GetSettings() [I] [63b02c54] [SQL] SELECT "id", "setting_key", "setting_value", "version", "created", "updated" FROM "system_setting" WHERE setting_key IN ($1) [picture.disable_gravatar] - 392.807µs
2022/12/31 13:34:31 ...ls/system/setting.go:121:GetSettings() [I] [63b02c54] [SQL] SELECT "id", "setting_key", "setting_value", "version", "created", "updated" FROM "system_setting" WHERE setting_key IN ($1) [picture.disable_gravatar] - 440.163µs
2022/12/31 13:34:31 ...ls/system/setting.go:121:GetSettings() [I] [63b02c54] [SQL] SELECT "id", "setting_key", "setting_value", "version", "created", "updated" FROM "system_setting" WHERE setting_key IN ($1) [picture.disable_gravatar] - 394.718µs
2022/12/31 13:34:31 ...ls/system/setting.go:121:GetSettings() [I] [63b02c54] [SQL] SELECT "id", "setting_key", "setting_value", "version", "created", "updated" FROM "system_setting" WHERE setting_key IN ($1) [picture.disable_gravatar] - 437.802µs
2022/12/31 13:34:31 ...ls/system/setting.go:121:GetSettings() [I] [63b02c54] [SQL] SELECT "id", "setting_key", "setting_value", "version", "created", "updated" FROM "system_setting" WHERE setting_key IN ($1) [picture.disable_gravatar] - 466.235µs
2022/12/31 13:34:31 ...ls/system/setting.go:121:GetSettings() [I] [63b02c54] [SQL] SELECT "id", "setting_key", "setting_value", "version", "created", "updated" FROM "system_setting" WHERE setting_key IN ($1) [picture.disable_gravatar] - 411.094µs
2022/12/31 13:34:31 ...ls/system/setting.go:121:GetSettings() [I] [63b02c54] [SQL] SELECT "id", "setting_key", "setting_value", "version", "created", "updated" FROM "system_setting" WHERE setting_key IN ($1) [picture.disable_gravatar] - 459.483µs
2022/12/31 13:34:31 ...ls/system/setting.go:121:GetSettings() [I] [63b02c54] [SQL] SELECT "id", "setting_key", "setting_value", "version", "created", "updated" FROM "system_setting" WHERE setting_key IN ($1) [picture.disable_gravatar] - 462.974µs
2022/12/31 13:34:31 ...ls/system/setting.go:121:GetSettings() [I] [63b02c54] [SQL] SELECT "id", "setting_key", "setting_value", "version", "created", "updated" FROM "system_setting" WHERE setting_key IN ($1) [picture.enable_federated_avatar] - 445.628µs
2022/12/31 13:34:31 ...ls/system/setting.go:121:GetSettings() [I] [63b02c54] [SQL] SELECT "id", "setting_key", "setting_value", "version", "created", "updated" FROM "system_setting" WHERE setting_key IN ($1) [picture.disable_gravatar] - 415.936µs
2022/12/31 13:34:31 ...ls/system/setting.go:121:GetSettings() [I] [63b02c54] [SQL] SELECT "id", "setting_key", "setting_value", "version", "created", "updated" FROM "system_setting" WHERE setting_key IN ($1) [picture.disable_gravatar] - 419.829µs
2022/12/31 13:34:31 ...ls/system/setting.go:121:GetSettings() [I] [63b02c54] [SQL] SELECT "id", "setting_key", "setting_value", "version", "created", "updated" FROM "system_setting" WHERE setting_key IN ($1) [picture.disable_gravatar] - 515.498µs
```

statement in the log.

### Gitea Version

1.18.0

### Can you reproduce the bug on the Gitea demo site?

No

### Git Version

2.39.0

### Operating System

Ubuntu 20.04.5

### How are you running Gitea?

Self hosted gitea-1.18.0-linux-amd64

### Database

PostgreSQL

**Comment by lunny** [comment]
I think this is duplicated with #22281 because you are using redis as cache?

**Comment by delvh** [comment]
Duplicate of #22281?

**Comment by somera** [comment]
> 

No, I'm using memcached

```
[cache]
ENABLED = true
ADAPTER = memcache
INTERVAL = 240
HOST = 192.168.178.30:11211
ITEM_TTL = 16h
```

**Comment by somera** [comment]
> I think this is duplicated with #22281 because you are using redis as cache?

No, I'm using memcached

```
[cache]
ENABLED = true
ADAPTER = memcache
INTERVAL = 240
HOST = 192.168.178.30:11211
ITEM_TTL = 16h
```

**Comment by somera** [comment]
The fix in 1.18.1 works. thx.
